my_project_test16742493171360845266_test_init/README.md

```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
```

Comment:
Did you mean to include this project at the root level of the repo?

Reply:
no, not at all. my bad 😵‍💫